src/wms/src/main/java/org/geoserver/wms/map/PDFMapResponse.java

```diff
@@ -117,153 +118,155 @@ public void write(Object value, OutputStream output, Operation operation)
             // height of document-object is height*72 inches
             com.lowagie.text.Rectangle pageSize = new com.lowagie.text.Rectangle(width, height);
```

Comment:
should this change? or do the packages remain the same despite the change?

Reply:
this class exists in openpdf: https://github.com/LibrePDF/OpenPDF/blob/master/openpdf/src/main/java/com/lowagie/text/Rectangle.java

Reply:
It looks like openpdf is forward compatible with all the old com.lowagie names. By contrast itextpdf moved to `com.itextpdf` when it went to v5+.